### ISSUE TITLE
Rasp Pi reboot required daily to access Habridge

### ISSUE THREAD

**Issue by Mooose1** [bwssytems/ha-bridge#1194]
I am not sure if this is a Win 10 problem, a RPi problem but here goes. Every time I reboot my desktop, I have to reboot the Pi (with Buster, HG and Habridge installed) to be able to access HG or Habridge through a browser window, or to be able to ssh into the pi with PuTTY. Any suggestions where to start??

**Comment by Mooose1** [comment]
A little more info. It seems that once a particular computer has had one ssh session with the Pi, every time that computer is restarted, it can no longer connect to the Pi, through the browser or through PuTTY.

**Comment by audiofreak9** [comment]
I've never experienced that, I ssh into my RPi often as it runs way more that just HA-Bridge.  Have you tried `$sudo systemctl status habridge.service -l`to see if the bridge is still running?

**Comment by Mooose1** [comment]
Yes, I have confirmed that HAbridge and Homegenie are still running. I also cannot start the ssh client until the Pi is rebooted. I had another complete RPi setup and logging in was never a problem after a shutdown, but that system would not work with Alexa. I have another laptop without the ssh client, and there are no issues there.

**Comment by beandi** [comment]
Sounds more like a network problem, maybe IP addresses are fixed and/or the dhcp server is somehow misconfigured - or you running out of available addresses?
